CONTRIBUTING.md

```diff
@@ -265,12 +264,41 @@ conflicts between slice definitions files;
 Apart from asserting your slice definitions' formatting, you must also test them
 before opening a Pull Request.
 
-Chisel supports custom releases, so please make sure you `cut` your custom
-slices and run smoke tests on them.
-
-***Tip**: to install slices from a custom Chisel release, simply provide its
-absolute path to the `--release` option in Chisel. E.g.
-`chisel cut --release $PWD/custom-chisel-release --root testfs pkg-a_sliceFoo`.*
+Every Chisel release support the functional testing for slices via
+[Spread](https://github.com/snapcore/spread).
+
+**We expect tests to be provided for every slice**, so please ensure you include
+those in your PRs.
+
+Here's how you can create and run tests for your slice definitions:
+
+1. under `tests/spread/integration/`, make sure at least one folder matching the
+package name you're slicing, exists,
+1. inside that folder you must create a `task.yaml` file. This is where you'll
+write all the tests for your slide definitions. Please follow the structure from
```

Comment:
Today I learnt: You can not number and just put 1 in an ordered list and markdown will take care of it!